ocfweb/static/js/pages/account/recommend_username.js

```diff
@@ -0,0 +1,19 @@
+function recommend() {
+    $.ajax({
+         type: 'GET',
+         url: 'recommend',
+         data: {'real_name': $('#real_name').html()},
+         success: function(data) {
+             $('#recommendations').empty();
+             recommendations = data['recommendations'];
+             for (var i in recommendations) {
+                 $('#recommendations').append(
+                     '<li class="list-group-item list-group-item-success">'+ recommendations[i] +'</li>'
```

Comment:
can we construct a jquery object here to avoid XSS vulnerabilities?

e.g. (untested, but i think this is right)

```javascript
$('recommendations').append(
    $('<li>')
        .addClass('list-group-item list-group-item-success')
        .text(recommendations[i])
);
```

also -- what if we made these recommendations clickable (click the recommendation to fill in the username box)?